sidebar.js

```diff
@@ -133,6 +133,20 @@ const sidebars = {
       items: [{ type: "autogenerated", dirName: "scripting/trust_scripts" }],
     },
   ],
+  jargonSidebar: [
+    {
+      type: "category",
+      label: "glossary",
+      collapsible: false,
+      link: {
+        type: "generated-index",
+        title: "Jargon",
+        description: "This is a list of all the jargons.",
+        slug: "jargon/glossary",
+      },
+      items: [{ type: "autogenerated", dirName: "jargon/glossary" }],
+    },
+  ],
 };
 
 module.exports = sidebars;
```

Comment:
edited label, title, description, slug and dirName to point to jargon when there's a right spot for it.

Reply:
We figured out a spot for it in #252 (and a discord conversation)
I'll add this to the PR.